SumOfArrays.kt

```diff
@@ -0,0 +1,31 @@
+import java.util.Arrays
+
+fun main(args: Array<String>) {
+    test_addNPrimes();
+}
+
+fun addNPrimes(possiblePrimes: Array<Int>, n: Int ): Int {
+    var pCount = 0;
+    var primes = possiblePrimes.filter { isPrime(it) && pCount++ < n }
```

Comment:
Since `n` is the number of primes the user wants to sum, a more fluid way to handle this would be with the `take` function. You won't need `pCount` at all:

```suggestion
    val primes = possiblePrimes.filter { isPrime(it) }.take(n)
```

Reply:
Thanks for the reviews! I will integrate these changed later tonight. This was my first time playing around with Kotlin. I really appreciate the advice on writing slicker code.